packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -138,29 +138,29 @@ To configure the client-based rate limit for existing orgs:
+Currently, a client-based rate limit only applies to an authorization server's OAuth 2.0 `/authorize` or `/login/login.htm` endpoint.
 
 **Q: How is the client-specific rate limit determined?**
 
-For the `/authorize` endpoint, the client rate limit framework calculates the per client rate limit based on the OAuth 2.0 client ID, the user's IP address, and the Okta device identifier (the Okta device identifier that Okta sets in the browser).
+For the OAuth 2.0 `/authorize` endpoint, the client rate limit framework calculates the per client rate limit based on the OAuth 2.0 client ID, the user's IP address, and the Okta device identifier (the Okta device identifier that Okta sets in the browser).
 
 For the `/login/login.htm` endpoint, the client rate limit framework calculates the per client rate limit based on the user's IP address and the Okta device identifier (the Okta device identifier that Okta sets in the browser).
 
 **Q: What happens if my network contains a proxy server through which the requests are proxied?**
 
-Requests would appear to come from the same IP Address. When `/authorize` or `/login/login.htm` requests are made from behind a proxy IP address, make sure to [configure the respective IPs as proxies](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_Network). This allows the client-based rate limit framework to look for the IP address before the proxy to find the true client IP address.
+Requests would appear to come from the same IP Address. When OAuth 2.0 `/authorize` or `/login/login.htm` requests are made from behind a proxy IP address, make sure to [configure the respective IPs as proxies](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_Network). This allows the client-based rate limit framework to look for the IP address before the proxy to find the true client IP address.
 
 **Q: Can I update the per client rate limit today?**
 
-No. Today every client ID/IP/device identifier combination (for the `/authorize` endpoint) or IP/device identifier combination (for the `/login/login.htm` endpoint) gets 60 total requests per minute and a maximum of two concurrent requests.
+No. Today every client ID/IP/device identifier combination (for the OAuth 2.0 `/authorize` endpoint) or IP/device identifier combination (for the `/login/login.htm` endpoint) gets 60 total requests per minute and a maximum of two concurrent requests.
 
 **Q: Does the org-wide rate limit still apply when I enable the client-based rate limit?**
 
 Yes. When the cumulative total request or maximum concurrent requests from every unique client exceeds the org-wide rate limits, your Okta org experiences org-wide rate limit errors.
 
-**Q: Would the rate limit headers returned by Okta on the /authorize or /login/login.htm endpoint reflect client-specific rate limits?**
+**Q: Would the rate limit headers returned by Okta on the /oauth2/v1/authorize or /login/login.htm endpoint reflect client-specific rate limits?**
```

Comment:
In this line and in a few others as well we have a specific version of the authorize endpoint listed, rather than the generic "OAuth 2.0 /authorize" mention that we're using elsewhere. Shouldn't it be the latter?

Reply:
Fixed.